pkg/machine/provider/platform.go

```diff
@@ -64,6 +64,26 @@ func InstalledProviders() ([]define.VMType, error) {
 	return []define.VMType{define.QemuVirt}, nil
 }
 
+func IsInstalled(provider define.VMType) (bool, error) {
+	switch provider {
+	case define.QemuVirt:
+		cfg, err := config.Default()
+		if err != nil {
+			return false, err
+		}
+		_, err = cfg.FindHelperBinary(qemu.QemuCommand, true)
+		if errors.Is(err, fs.ErrNotExist) {
+			return false, nil
+		}
+		if err != nil {
+			return false, err
+		}
+		return true, nil
+	default:
+		return false, nil
+	}
+}
```

Comment:
I feel like these are similar in both functionality and code to the `InstalledProviders` functions that are implemented already. I think that it would be best to have either one or the other but not both.